README.md

```diff
@@ -42,6 +42,10 @@ If you want to use your own push or other customizations:
 
 ## Inputs
 
+### user
+
+The user name for the token. The default is Bearer, which works with GitHub packages tokens. For other sites you may need a user name or set to empty string to just use a token (e.g. rubygems).
+
 ### token
 
 The token to be used for authenticating with GitHub packages. If you are pushing a gem to the same repo with a matching name, the default token is enough:
```

Comment:
I reckon we should add something about the RubyGems API key here

Reply:
I do as well, this was a quick fix for hermod, but needs better docs. Maybe even a better v3 interface! e.g.

```yaml
    - uses: fac/ruby-gem-setup-credentials-action@v3
      with:
        rubygems: ${{ secrets.FAC_RUBYGEMS_KEY }}
    - uses: fac/ruby-gem-setup-credentials-action@v3
      with:
        github: ${{ secrets.github_token }}
```

Reply:
Extra doc added.